{"name": "Авторизация без поля password", "status": "passed", "description": "Проверка, что без password запрос возвращает ошибку 400", "steps": [{"name": "Логин курьера", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"/Users/kostya/PycharmProjects/Sprint7/venv/lib/python3.10/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/Users/kostya/PycharmProjects/Sprint7/api/courier_api.py\", line 31, in login_courier\n    return requests.post(COURIER_LOGIN, json=payload)\n  File \"/Users/kostya/PycharmProjects/Sprint7/venv/lib/python3.10/site-packages/requests/api.py\", line 115, in post\n    return request(\"post\", url, data=data, json=json, **kwargs)\n  File \"/Users/kostya/PycharmProjects/Sprint7/venv/lib/python3.10/site-packages/requests/api.py\", line 59, in request\n    return session.request(method=method, url=url, **kwargs)\n  File \"/Users/kostya/PycharmProjects/Sprint7/venv/lib/python3.10/site-packages/requests/sessions.py\", line 589, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"/Users/kostya/PycharmProjects/Sprint7/venv/lib/python3.10/site-packages/requests/sessions.py\", line 703, in send\n    r = adapter.send(request, **kwargs)\n  File \"/Users/kostya/PycharmProjects/Sprint7/venv/lib/python3.10/site-packages/requests/adapters.py\", line 486, in send\n    resp = conn.urlopen(\n  File \"/Users/kostya/PycharmProjects/Sprint7/venv/lib/python3.10/site-packages/urllib3/connectionpool.py\", line 787, in urlopen\n    response = self._make_request(\n  File \"/Users/kostya/PycharmProjects/Sprint7/venv/lib/python3.10/site-packages/urllib3/connectionpool.py\", line 534, in _make_request\n    response = conn.getresponse()\n  File \"/Users/kostya/PycharmProjects/Sprint7/venv/lib/python3.10/site-packages/urllib3/connection.py\", line 565, in getresponse\n    httplib_response = super().getresponse()\n  File \"/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/http/client.py\", line 1374, in getresponse\n    response.begin()\n  File \"/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/http/client.py\", line 318, in begin\n    version, status, reason = self._read_status()\n  File \"/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/http/client.py\", line 279, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/socket.py\", line 705, in readinto\n    return self._sock.recv_into(b)\n  File \"/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/ssl.py\", line 1274, in recv_into\n    return self.read(nbytes, buffer)\n  File \"/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/ssl.py\", line 1130, in read\n    return self._sslobj.read(len, buffer)\n"}, "parameters": [{"name": "payload", "value": "{'login': 'hkhcuxfclo'}"}], "start": 1761159819006, "stop": 1761159869729}], "start": 1761159819006, "stop": 1761159869739, "uuid": "dd3515fc-44d7-4a99-87c6-2789d3b3a16e", "testCaseId": "36c66187d279e2cd63e9b7521c9f8cce", "fullName": "tests.test_courier_login.TestCourierLogin#test_login_without_password_fails"}